community/asset_library/submitting_to_assetlib.rst

```diff
@@ -75,6 +75,28 @@ library a better place for all users.
 * If your asset is a library for working with other files,
   consider including example files in the asset.
 
+* Consider adding a **.gitattributes** file to your repo and listing
+  files not required by your asset to function with the ``export-ignore``
+  directive. This directive removes such files from the resulting ZIP file
+  and prevents them from being downloaded by the asset library users.
+  For a typical plugin **.gitattributes** may look like this:
+
+  .. code-block:: none
+
+    /.gitattributes     export-ignore
+    /.gitignore         export-ignore
+    /LICENSE            export-ignore
+    /README.md          export-ignore
+    /project.godot      export-ignore
```

Comment:
If we are excluding `project.godot`, I would also exclude the icon files which are generally `icon.png` or `icon.svg` (it's fine to have nonexistent paths in `.gitattributes`).

I would also exclude the license file if it has a `.md` extension:

```suggestion
    /.gitattributes     export-ignore
    /.gitignore         export-ignore
    /LICENSE            export-ignore
    /LICENSE.md         export-ignore
    /README.md          export-ignore
    /project.godot      export-ignore
    /icon.png           export-ignore
    /icon.svg           export-ignore
```

Reply:
Seems redundant if you don't actually have those files, but can be added I guess.